### ISSUE TITLE
[Shipping] Shipping class dropdown

### ISSUE THREAD

**Issue by jarekmorawski** [woothemes/woocommerce#34328]
**User story**

- As a merchant,
- I want to enter accurate shipping information
- So that I don’t overcharge my customers

The create new shipping class logic will be done as part of https://github.com/woocommerce/woocommerce/issues/34657

**Design**
5sAIeTRd9Yp7nSCT33BAWz-fi-6499%3A290914

**Acceptance criteria**

- [x] Users can view and select any existing shipping class.
- [x] In addition to user-created shipping classes, it also contains additional options: `No shipping class` / `Standard shipping`
- [x] The field’s helper text link opens `WooCommerce > Settings > Shipping > Shipping classes` in a new tab.

**Comment by mdperez86** [comment]
> In addition to user-created shipping classes, it also contains additional options: `No shipping class` / `Standard shipping`

Hello @jarekmorawski and  @pmcpinto 
What is the default selected option?
Since `Standard shipping` is not a real shipping class option, I mean, it does not exists in the database as a shipping class record by default. What should be the behavior if the user select it and then save the product?

**Comment by jarekmorawski** [comment]
Hiya! Thanks for the ping. It's a great question because I actually thought we had a default shipping class. In my test store, it's called Standard shipping, and I don't think I've created it manually (maybe some plugin did).

> What is the default selected option?

The default should be `No shipping class`. For the MVP, we can leave it at that, but in the future, we could add a second option to the dropdown `New shipping class` that would open a modal. I'm not sure if we can squeeze this into this cycle, but just in case, I added the design here: `5sAIeTRd9Yp7nSCT33BAWz-fi-6590%3A287899`

> What should be the behavior if the user select it and then save the product?

No change. If the default is `No shipping class`, it'd be the same before and after saving the product.
